packages/react-components/react-motion-preview/src/hooks/useMotion.ts

```diff
@@ -177,11 +177,11 @@ function useMotionPresence<Element extends HTMLElement>(
       ref,
       type,
       active,
-      canRender: type !== 'unmounted',
+      canRender: presence || type !== 'unmounted',
```

Comment:
can we add test that demonstrates behaviour / the fix respectively ?  from a person not involved I can only guess what is this about. ty